includes/User.php

```diff
@@ -4927,52 +4921,53 @@ protected function loadOptions() {
 		wfRunHooks( 'UserLoadOptions', array( $this, &$this->mOptions ) );
 	}
 
-	private function loadAttributes() {
-		global $wgEnableReadsFromAttributeService;
-
-		if ( !empty( $wgEnableReadsFromAttributeService ) ) {
-			$attributes = $this->userAttributes()->getAttributes($this->getId());
-			foreach ( $attributes as $attributeName => $attributeValue ) {
-				$this->compareAttributeValueFromService( $attributeName, $attributeValue );
-
-				 $this->mOptionOverrides[$attributeName] = $attributeValue;
-				 $this->mOptions[$attributeName] = $attributeValue;
-			}
-		}
+	/**
+	 * Save this user's preferences into the database.
+	 *
+	 * @see getGlobalPreference for documentation about preferences
+	 */
+	protected function savePreferences() {
```

Comment:
Git is getting confused here. I didn't add this `savePreferences()` method, it existed before. You can see git thinks I deleted it [here](https://github.com/Wikia/app/pull/8792/files#diff-897fba1dd6b8edc3238762e35d0b7c71L4974) and then recreated it here.